src/cli/index.js

```diff
@@ -333,8 +333,8 @@ config.init({
       logError(errs);
     }
 
-    if (commands[commandName]) {
-      reporter.info(getDocsInfo(commandName));
+    if (commands[commandName] || aliases[commandName]) {
+      reporter.info(getDocsInfo(commands[commandName] || aliases[commandName]));
```

Comment:
instead of doing this twice, can we do:

```
const helpCommand = commands[commandName] || aliases[commandName];
if (helpCommand) {
  reporter.info(getDocsInfo(helpCommand));
}
```

helpCommand might not be the best name. Maybe `realCommandName`?